52 - ホーム画面のレイアウトを変更

racode246 opened the pull request:
## 説明
ナビゲーションバーを変更
ホーム画面を削除

## issue
#52

## 備考
背景、ボタンのスタイルなどは他のissueが担当してるのでここでは変更していない

racode246 committed:
## 説明
ナビゲーションバーを変更
ホーム画面を削除

## issue
#52

## 備考
背景、ボタンのスタイルなどは他のissueが担当してるのでここでは変更していない

Dimda commented on src/components/Router.js:
```diff
@@ -4,11 +4,11 @@ import { Spinner } from './common';
 import ListArticlesContainer from '../containers/ListArticlesContainer';
 import SearchResultsContainer from '../containers/SearchResultsContainer';
 import firebase from 'firebase';
+import { IconWrapper } from './common/IconWrapper';
```
`common`の中に入っている`index.js`で`IconWrapper`を追加したい。そうしたらimportする時に`'./common/IconWrapper'`じゃなくて、`./common`だけで複数のコンポーネントをimportできるようになる。

Dimda commented on src/components/Router.js:
```diff
@@ -19,15 +19,54 @@ const RouterComponent = ({ pageMoved }) => (
-        <Scene key='listarticles' component={ListArticlesContainer} title='記事一覧' />
-        <Scene key='searchresults' component={SearchResultsContainer} title='検索結果' />
+      <Scene key='main' tabs showLabel={false}>
+        <Scene
```
`hideNavBar={true}`を追加したらnavbarが消える

racode246 replied:
navBarがいらないのは検索ページだけだから、ここだけ適用する

Dimda commented on src/components/Router.js:
```diff
@@ -19,15 +19,54 @@ const RouterComponent = ({ pageMoved }) => (
+          type='simple-line-icon'
+          name='magnifier'
+        />
+        <Scene
```
`hideNavBar={true}`を追加したらnavbarが消える

Dimda commented on src/components/Router.js:
```diff
@@ -19,15 +19,54 @@ const RouterComponent = ({ pageMoved }) => (
+          type='simple-line-icon'
+          name='note'
+        />
+        <Scene
```
`hideNavBar={true}`を追加したらnavbarが消える

Dimda commented on src/components/Router.js:
```diff
@@ -19,15 +19,54 @@ const RouterComponent = ({ pageMoved }) => (
+          type='font-awesome'
+          name='bookmark-o'
+        />
+        <Scene
```
`hideNavBar={true}`を追加したらnavbarが消える

racode246 committed:
52 - ホーム画面のレイアウトを変更

## 説明
common/index.jsにexportを追加
サーチページのナビゲーションを削除
サーチページ以外のページにタイトルを追加

## issue
#52

racode246 committed:
Merge branch 'master' into feature/52-update-home-screen-layout

Dimda reviewed (approved):
大丈夫と思う。 👍 
注意するのは一番目のコミットの書き方は違うとできれば`pull`でマージをするより`rebase`でやって欲しい。そっちの方で履歴が綺麗になると思う。
タブバー導入で挙動がおかしいところあるけどそれのために別のタスクカードを用意する。
